man/subuid.5.xml

```diff
@@ -48,11 +53,37 @@
   </refmeta>
   <refnamediv id='name'>
     <refname>subuid</refname>
-    <refpurpose>the subordinate uid file</refpurpose>
+    <refpurpose>the configuration for subordinate user ids</refpurpose>
   </refnamediv>
 
   <refsect1 id='description'>
     <title>DESCRIPTION</title>
+    <para>
+      Subuid authorizes a user id to map lists of user ids from its namespace
+      into child namespaces. This allows the user from the parent namespace to
+      use the user ids from the child.
```

Comment:
I'd drop the second sentence and maybe not speak of "lists" but rather for "ranges"?:

"Subid authorizes a user id to map ranges of user ids from its namespace into child namespaces."

Reply:
Fixed and also changed the sentence in `subgid.5.xml`